docs/pages/database-access/guides/rds-proxy.mdx

```diff
@@ -0,0 +1,228 @@
+- Any RDS Proxies intended for connection through Teleport must have TLS enabled.
+- A host, e.g., an EC2 instance, where you will run the Teleport Database
+  Service.
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create a Database Service configuration
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Create the Database Service configuration:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=teleport.example.com:443 \
+   --token=/tmp/token \
+   --rdsproxy-discovery=us-west-1
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=mytenant.teleport.sh:443 \
+   --token=/tmp/token \
+   --rdsproxy-discovery=us-west-1
+```
+
+</ScopedBlock>
+
+The command will generate a Database Service configuration with RDS Proxies
+auto-discovery enabled on the `us-west-1` region and place it at the
+`/etc/teleport.yaml` location.
+
+## Step 2/5. Create an IAM policy for Teleport
+
+Teleport needs AWS IAM permissions to be able to discover and register RDS
+Proxies.
+
+(!docs/pages/includes/database-access/aws-bootstrap.mdx!)
+
+## Step 3/5. Start the Database Service
+
+Start the Database Service:
+
+```code
+$ teleport start --config=/etc/teleport.yaml
+```
+
+The Database Service will discover all RDS Proxies according to the
+configuration and register them in the cluster. In addition to the primary
+endpoints of the RDS Proxies, their custom endpoints will also be registered.
+
+## Step 4/5. Configure database user credentials
+
+The Database Service connects to a RDS Proxy instance using IAM authentication.
+In addition, the RDS Proxy instance must also be able to connect to the RDS
+DB instance or Aurora DB cluster using pre-configured database user credentials.
+
+First use Secrets Manager to store sets of user name and password credentials.
+You create a separate Secrets Manager secret for each database user account
+that RDS Proxy connects to on the RDS DB instance or Aurora DB cluster. The
+password associated with the secret must match the database password for that
+user in the target database.
+
+For example, the following AWS CLI command creates a Secrets Manager secret for
+database user `alice`:
+
+```
+aws secretsmanager create-secret \
+  --name rdsproxy_alice --description "database user alice" \
+  --secret-string '{"username":"alice","password":"password_for_alice"}'
+```
+
+Next, create an IAM role with a policy that can access these secrets, for
+example:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "secretsmanager:GetSecretValue",
+            "Resource": [
+                "arn:aws:secretsmanager:us-west-1:account_id:secret:rdsproxy_alice",
```

Comment:
Just flagging this as another reason to prioritize https://github.com/gravitational/docs/issues/186, as we could replace `alice` with a Var everywhere else but here.